.circleci/config.yml

```diff
@@ -205,20 +179,20 @@ commands:
                   fi
                 done
                 if [[ $changed = false ]]; then
-                  echo "Code in '<< parameters.dir_names >>' directories are not changed"
+                  echo "Code in '<< parameters.dir_names >>' directories are not changed. Exiting job."
                   circleci-agent step halt
                 fi
               fi
             fi
-          name: Halt if code in '<< parameters.dir_names >>' directories are not changed
+          name: "Halt job if code in '<< parameters.dir_names >>' directories are not changed"
 
   launch-local-api:
     description: "Launch local API server"
     steps:
       - run:
           # Watch the port for 2m for MySQL start up
           name: Await MySQL startup
-          command: dockerize -wait tcp://127.0.0.1:3306 -timeout 2m
+          command: dockerize -wait http://127.0.0.1:3306 -timeout 2m
```

Comment:
Pretty sure the MySQL connection is not http; did this actually work?

Reply:
I don't know. it's not used by any job yet. I will revert it back to tcp. I was doing Find & Replace. :(